src/components/shop/shop.js

```diff
@@ -81,7 +81,10 @@ export class Shop extends Screen {
 
     setVisiblePane(pane) {
         Object.keys(this.panes).forEach(key => this.panes[key] && this.panes[key].setVisible(pane === key));
-        pane === "top" && this.title.setTitleText("Shop");
+        if (pane === "top") {
+            gmi.setStatsScreen("shopmenu");
```

Comment:
do we want this stat every time you hit the top, or only when you first enter the shop?

Reply:
Every time we hit the top, in addition to the 'free' stat we get from screen.js.